cpp/core/config/GlutenConfig.h

```diff
@@ -17,10 +17,13 @@
 
 #pragma once
 
+#include <jni.h>
```

Comment:
Is this needed?

Reply:
Yes, because the `getConfMap` need `JNIEnv`.